### PR TITLE
ILS: Report runner view

### PR DESCRIPTION
**Pull request by kkrampa**
@czue I need this views to do some cleanup in runner in ILS domain

### CODE REVIEW

**Comment by kkrampa** [comment]
@czue Any chance to have this merged today?

File: custom/ilsgateway/views.py [hunk 1]
```diff
@@ -328,3 +331,29 @@ def change_runner_date_to_last_migration(request, domain):
     last_run.end = checkpoint.date
     last_run.save()
     return HttpResponse('OK')
+
+
+class ReportRunListView(ListView, DomainViewMixin):
```
Comment (czue) [inline custom/ilsgateway/views.py]: do these views need any permission markup on them?

File: custom/ilsgateway/views.py [hunk 2]
```diff
@@ -328,3 +331,29 @@ def change_runner_date_to_last_migration(request, domain):
...
+            raise Http404()
+        return super(ReportRunListView, self).dispatch(request, *args, **kwargs)
+
+    def get_queryset(self):
+        return ReportRun.objects.filter(domain=self.domain).order_by('pk')
+
+
+class ReportRunDeleteView(DeleteView, DomainViewMixin):
+    model = ReportRun
+    success_url = reverse_lazy('report_run_list')
```
Comment (czue) [inline custom/ilsgateway/views.py]: how does this work without the domain?

Comment (czue) [inline custom/ilsgateway/views.py]: or i guess maybe a better question - does this do anything?

Comment (kkrampa) [inline custom/ilsgateway/views.py]: Right, deleted

File: custom/ilsgateway/views.py (hunk 1 above)
Comment (kkrampa) [inline custom/ilsgateway/views.py]: Added in dispatcher: https://github.com/dimagi/commcare-hq/pull/7375/files#diff-774201133ac94fd25b946f8c1eb46c3cR341

**Comment by czue** [comment]
cool, doesn't look like any test coverage so merging

**Pull request merged**